integrations/amplitude/lib/index.js

```diff
@@ -392,6 +393,31 @@ Amplitude.prototype.setRevenue = function(properties) {
   }
 };
 
+// equivalent of sending includeReferrer=true in the amplitude config object when we initialize
+Amplitude.prototype.sendReferrer = function() {
```

Comment:
This aims to capture the exact same logic as the native Amplitude logic [here](https://github.com/amplitude/Amplitude-JavaScript/blob/8f9cede18877676315a6def831a9383f3e990374/src/amplitude-client.js#L523-L538)